src/traccuracy/track_errors/_ctc.py

```diff
@@ -80,12 +80,16 @@ def get_edge_errors(matched_data: "Matched"):
         logger.info("Edge errors already calculated. Skipping graph annotation")
         return
 
+    # Node errors must already be annotated
+    if not comp_graph.node_errors and not gt_graph.node_errors:
+        logger.warning("Node errors have not been annotated. Running node annotation.")
+        get_vertex_errors(matched_data)
+
     induced_graph = comp_graph.get_subgraph(
         comp_graph.get_nodes_with_flag(NodeAttr.TRUE_POS)
     ).graph
 
     comp_graph.set_edge_attribute(list(comp_graph.edges()), EdgeAttr.FALSE_POS, False)
-    comp_graph.set_edge_attribute(list(comp_graph.edges()), EdgeAttr.TRUE_POS, False)
```

Comment:
@msschwartz21 I don't think I fully understand why we are not annotating true positive edges? You mention in the description that we can't rely on the assumption that `TP` and `WS` edges are mutually exclusive, but I think we can? I don't think a `TP` edge should ever be annotated with `WS`.

Reply:
My stab at explaining this, that Morgan can elaborate on at her leisure: I admit that I am always confused by`WS` edges; however, it is my understanding that if there is a division and we correctly detect one of the edges to a daughter but not the other, the one we correctly detected would be labeled `WS` - correct? I would consider that a `TP` - we correctly identified the edge! Just because we missed the other daughter edge (a `FN` edge), that doesn't make the correctly recovered edge incorrect. In that way, something that is a `WS` edge in the CTC metric could be considered a `TP` elsewhere, making them not mutually exclusive.

Reply:
^ Caroline's description captures my thoughts perfectly